.gitlab-ci.yml

```diff
@@ -95,8 +95,6 @@ upload-artifacts:
     - ln -s "$CI_OWNCLOUD_PASSWORD" ~/.netrc
     - sed -i 's/^/machine ftp.essensium.com login prplmesh-robot-ci password /' ~/.netrc
     - ci/upload_to_owncloud.sh -v --overwrite-remote-path "artifacts/$CI_JOB_ID" artifacts/latest ./build
-  rules:
-    - if: $CI_COMMIT_BRANCH == "master"
```

Comment:
Nice to have this commit for testing. However, you should have made sure it doesn't have your Sob. That way, the DCO check will fail and you can be sure it won't get merged accidentally.